src/components/ChartView/ChartVersionsList.tsx

```diff
@@ -1,22 +1,47 @@
 import * as React from "react";
+import { Link } from "react-router-dom";
 
 import { IChartVersion } from "../../shared/types";
+import * as url from "../../shared/url";
 
 interface IChartVersionsListProps {
+  selected: IChartVersion;
   versions: IChartVersion[];
 }
 
-class ChartVersionsList extends React.Component<IChartVersionsListProps> {
+interface IChartVersionsListState {
+  showAll: boolean;
+}
+
+class ChartVersionsList extends React.Component<IChartVersionsListProps, IChartVersionsListState> {
+  public state: IChartVersionsListState = {
+    showAll: false,
```

Comment:
aside: we probably need to come up with some guidelines for when to use component state vs redux state. IMO if we're doing anything that doesn't mutate app data and isn't shared, it's better to use component state.